source/Lib/CommonLib/x86/InitX86.cpp

```diff
@@ -125,10 +125,11 @@ void RdCost::initRdCostX86()
   switch (vext){
     case AVX512:
     case AVX2:
-#ifndef REAL_TARGET_WASM
+#if !defined( REAL_TARGET_WASM ) && defined( _MSC_VER ) && _MSC_VER >= 1938
```

Comment:
This condition is incorrect for the `REAL_TARGET_WASM` case.
You probably meant
```
#if defined( REAL_TARGET_WASM ) || (defined( _MSC_VER ) && _MSC_VER >= 1938)
```